packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js

```diff
@@ -143,13 +143,11 @@ export default class OvhManagerNetAppNetworkConfigurationCtrl {
           if (!data.vrackAssociationStatus?.success) {
             this.Alerter.set(
               'alert-warning',
-              this.$translate.instant(
+              `${this.$translate.instant(
                 'netapp_network_configuration_vrack_association_warning',
-                {
-                  message: data.vrackAssociationStatus.message,
-                  requestId: data.vrackAssociationStatus.requestId,
-                },
-              ),
+              )}<br> ${data.vrackAssociationStatus.message}<br> request_id: ${
```

Comment:
```suggestion
              )}<br /> ${data.vrackAssociationStatus.message}<br> request_id: ${
```

Reply:
There is no need to close sinlge tags in html 5 : https://www.w3schools.com/tags/tag_br.asp